data/strings/strings.txt

```diff
@@ -20600,6 +21069,7 @@
     ja = 検索を使うか、地図上をタップして目的地を追加する
     ko = 검색을 사용하거나 지도를 탭하여 목적지 지점을 추가합니다
     lt = Naudokite paiešką arba bakstelėkite žemėlapyje, kad pridėtumėte paskirties tašką
+    lv = Lietojiet meklēšanu vai uzsitiet uz kartes, lai pievienotu galamērķi
```

Comment:
"uzsitiet" -> "pieskarieties"

Reply:
And then also "uz kartes" -> "kartei".

Reply:
Yes, and thanks —
"Lietojiet meklēšanu vai pieskarieties kartei, lai pievienotu galamērķi"